tests/Provider/QueryBuilderProvider.php

```diff
@@ -171,11 +171,11 @@ public static function batchInsert(): array
                 [['no columns passed']],
                 'expected' => DbHelper::replaceQuotes(
                     <<<SQL
-                    INSERT INTO [[customer]] () VALUES (:qp0)
+                    INSERT INTO [[customer]] ([[id]]) VALUES (:qp0)
                     SQL,
                     static::$driverName,
                 ),
-                'expectedParams' => [':qp0' => 'no columns passed'],
+                'expectedParams' => [':qp0' => 0],
```

Comment:
Why 'no columns passed' casted to int ?

Reply:
The first column from the table `customer` is `id` and it is `int`